frontend/ui/elements/physical_buttons.lua

```diff
@@ -49,6 +49,17 @@ if Device:hasDPad() and Device:useDPadAsActionKeys() then
             G_reader_settings:flipNilOrFalse("input_invert_right_page_turn_keys")
             Device:invertButtonsRight()
         end,
+        separator = true,
+    })
+    table.insert(PhysicalButtons.sub_item_table, {
+        text = _("Use left and right keys for page turning"),
+        checked_func = function()
+            return G_reader_settings:isTrue("left_right_turn_pages")
```

Comment:
The other setting names in this file have the word "key" in them. This one should too.